TestApp/android/app/src/main/java/com/testapp/MainApplication.java

```diff
@@ -37,19 +26,15 @@ public boolean getUseDeveloperSupport() {
 
         @Override
         protected List<ReactPackage> getPackages() {
-            return Arrays.asList(
-                    new MainReactPackage(),
-                    new RNGestureHandlerPackage(),
-                    new TestAppNativePackage(getApplication()),
-                    new AppCenterReactNativeAuthPackage(getApplication()),
```

Comment:
Currently we have a limitation that `AppCenterReactNativeAuthPackage` must be started before `AppCenterReactNativeDataPackage` so the order matters. 

We will need to figure out if there is a way to keep the order with autolinking.

Reply:
Why do we have this limitation?

Reply:
We don't know when one will remove auth module. Core channel blocks on refreshing token from auth, so if SDK started without auth, channel will sign out assuming auth module was removed.
React-native splits start modules also.